apps/arweave/src/ar_sync_record.erl

```diff
@@ -201,6 +202,17 @@ get_next_synced_interval(Offset, RightBound, ID, StoreID) ->
 			ar_ets_intervals:get_next_interval(TID, Offset, RightBound)
 	end.
 
+%% @doc Return the lowest unsynced interval with the end offset strictly above the given Offset
+%% and with the right bound at most RightBound.
```

Comment:
What's the difference between "end offset" and "right bound"? I may have the layout flipped in my head, but I would have assumed they are the same?

Reply:
Renamed it to [end offset upper bound](https://github.com/ArweaveTeam/arweave/pull/556/commits/9bff8968d2fb4c66faf098743cfc04a04b36cc1a).